core/controllers/skill_editor.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for the skill editor."""
+from core.controllers import base
+from core.domain import acl_decorators
+from core.domain import skill_domain
+from core.domain import skill_services
+import feconf
+import utils
+
+
+def _require_valid_skill_id(skill_id):
```

Comment:
put this in skill_domain.Skill as `@classmethod` and add unit tests

See https://github.com/oppia/oppia/pull/4952/files#diff-f539af1da991345c436a49a41821b7e3R820 and similar comment [here](https://github.com/oppia/oppia/pull/4952#discussion_r192573612).

Reply:
Done.